training/offline/online_eval.py

```diff
@@ -105,7 +106,20 @@ def main(args):
     os.makedirs(exp_dir, exist_ok=True)
 
     if args.ckptStep is None:
-        raise ValueError("ckptStep is None")
+        query = os.path.join(ckpt_dir, "*.ckpt")
+        print(query)
+        found = glob.glob(query)
+
+        if len(found) == 0:
+            query = os.path.join(args.output_basedir, args.training_run_id, "*.ckpt")
+            print(query)
+            found = glob.glob(query)
+
+        if len(found) == 0:
+            raise ValueError("No checkpoints found.")
+
+        ckpt_pth = sorted(found)[-1]
+        print(f"No checkpoint step given. Using {ckpt_pth}.")
```

Comment:
same here. I found that both of you are trying to get the local evaluation working xD